lib/game_collection.rb

```diff
@@ -17,7 +17,27 @@ def create_games(file_path)
     end
   end
 
+
+  def create_pct_data
+    @games_list.each do |game|
+      @pct_data[:total_games] += 1
+      if game.home_goals == game.away_goals
+        @pct_data[:ties] += 1
+      elsif game.home_goals > game.away_goals
+        @pct_data[:home_wins] += 1
+      else
+        @pct_data[:away_wins] += 1
+      end
+    end
+    @pct_data
+  end
```

Comment:
this method will calculate three percentages for iteration 2. methods 4, 5, and 6